src/radical/pilot/agent/lm/orte.py

```diff
@@ -262,12 +260,14 @@ def construct_command(self, cu, launch_script_hop):
             node_id = node['uid'].rsplit('_', 1)[-1] 
 
             # add all cpu and gpu process slots to the node list.
+            # pylint: disable=unused-variable
             for cpu_slot in node['core_map']: hosts_string += '%s,' % node_id
             for gpu_slot in node['gpu_map' ]: hosts_string += '%s,' % node_id
             for cpu_slot in node['core_map']: depths.add(len(cpu_slot))
+            # pylint: enable=unused-variable
```

Comment:
The unused variable should be replaced by `_`.  We don't need a linter if we are not using it ;-)